cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -57,6 +57,9 @@ pub enum Inst {
         loc: SourceLoc,
     },
 
+    /// The high result of a (un)signed multiply: imul/mul using RAX:RDX.
```

Comment:
Could you clarify the register usage in this comment a bit?  Something like: `RDX:RAX <- RAX * rhs`.